frameworks/Crystal/moonshine/setup-postgres.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-fw_depends crystal
+curl http://dist.crystal-lang.org/apt/setup.sh | sudo bash
+
+sudo apt-get install crystal
```

Comment:
Hi, Kain.

I did not know all the framework should need to use fw_depends structure
or whatever that function is.

What I hear from you guys was just fix problem.

I fixed this issue and now crystal/moonshine uses fw_depends function to
setup environments.

Everything is good now.

On 12/21/15 11:17 AM, Mike Smith wrote:
> In frameworks/Crystal/moonshine/setup-postgres.sh
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1813#discussion_r48179586>:
> 
>> @@ -1,6 +1,8 @@
>>  #!/bin/bash
>>  
>> -fw_depends crystal
>> +curl http://dist.crystal-lang.org/apt/setup.sh | sudo bash
>> +
>> +sudo apt-get install crystal
> 
> These lines are 100% wrong - use the |fw_depends| structure in all setup
> files. If these are the correct steps (and I am not saying that they
> are), then they should be in this file
> <https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/languages/crystal.sh>,
> not in setup files like this one.
> 
> EDIT: Additionally, we are striving to moving away from |apt-get|, which
> is why this file
> <https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/setup/linux/languages/crystal.sh>
> does not use it.
> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TechEmpower/FrameworkBenchmarks/pull/1813/files#r48179586>.
> 

-- 
Best,
Byung Lee